### DISCUSSION TITLE
开启alist的登陆，alist服务设置webdav的密码WEBDAV_PASSWORD

### DISCUSSION THREAD

**Discussion by kint7669** [monlor/docker-xiaoya#287] [General]
开启alist的登陆，alist服务设置webdav的密码WEBDAV_PASSWORD

WEBDAV_PASSWORD=123456

设置好后重启过容器，

好像完全没有效果哦？还是不需要密码就可以访问。

**Comment by monlor** [comment]
不能直接重启，需要reload，或者docker compose up -d

**Reply by kint7669** [reply]
感谢，可以了。
